input/pagecontent/aucdi.md

```diff
@@ -157,64 +157,57 @@ Column attribute descriptions are as follows:
   </tr>
     <tr>
     <td colspan="2">Last updated</td>
-    <td>TBD</td>
-    <td>TBD</td>
-    <td>This data element is added to AUCDI R1; work is underway to map to AU Core.</td>
+    <td>-</td>
+    <td>-</td>
+    <td>This AUCDI element is a <a href="future.html#future-candidate-requirements-under-consideration">Future Candidate Requirement Under Consideration</a>. Future versions of AU Core will develop and test approaches to addressing Last Updated.</td>
   </tr>
   <tr>
     <td rowspan="9">Medication use statement</td>
     <td colspan="2">Medication name</td>
     <td>AU Core MedicationStatement | <a href="StructureDefinition-au-core-medication.html">AU Core Medication</a></td>
     <td>MedicationStatement.medication[x] | Medication.code</td>
-    <td>Development and testing of AU Core MedicationStatement profile is planned for AU Core R2.</td>
+    <td rowspan="8"><p>Development and testing of AU Core MedicationStatement profile is planned for AU Core R2.</p><p>See <a href="medicine-information.html">Medicine Information Guidance</a>.</p></td>
```

Comment:
Suggest changing the hyperlink text to match the page title for consistency.

Reply:
The page title is Medicine Information. But If you are concerned on having the word guidance inside the hyperlink I'll change to that for now. I think it's better with it in.